### ISSUE TITLE
accessibility audit, tooling

### ISSUE THREAD

**Issue by darrenjennings** [Kong/kongponents#174]
A11y (accessibility) was unfortunately not a focus when getting these components up, but it doesn't have to stay that way. Not sure best way to audit in an automated fashion, so if someone has recommended resources - would be helpful. Here's what I found with a quick google:

https://github.com/vue-a11y/vue-axe - vue plugin to print a11y warnings to the console
https://github.com/maranran/eslint-plugin-vue-a11y - adds eslinting a11y rules to SFCs

**Comment by hutchic** [comment]
https://www.w3.org/WAI/ER/tools/ has a fairly comprehensive list

**Comment by xitongzou** [comment]
created ticket: https://konghq.atlassian.net/browse/INTF-2494
